<launch>
     <node name="tf2_web_republisher" pkg="tf2_web_republisher" type="tf2_web_republisher" respawn="true"/>	
     <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>		
     <node name="mvibot_server" pkg="mvibot_v4" type="mvibot_serverv3" respawn="true"/>
</launch>	
